app/src/main/java/org/sopt/alami/presentation/main/MainScreen.kt

```diff
@@ -0,0 +1,40 @@
+package org.sopt.alami.presentation.main
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.material3.Scaffold
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import kotlinx.collections.immutable.toPersistentList
+import org.sopt.alami.core.designsystem.theme.AlarmiTheme
+import org.sopt.alami.presentation.main.component.MainBottomBar
+
+@Composable
+fun MainScreen(
+    navigator: MainNavigator = rememberMainNavigator()
+) {
+    Scaffold(
+        modifier = Modifier
+            .fillMaxSize(),
+
+        bottomBar = {
+            MainBottomBar(
+                modifier = Modifier
+                    .background(AlarmiTheme.colors.grey800),
+                isVisible = navigator.showBottomNavigator(),
+                tabs = MainTabType.entries.toPersistentList(),
+                currentTab = navigator.currentTab,
+                onTabSelected = navigator::navigate
+
+            )
+        },
+
+        content = { padding ->
+            MainNavHost(
+                navigator = navigator,
+                padding = padding,
+                modifier = Modifier
+            )
+        }
+    )
+}
```

Comment:
p1
ㅎㅎ후행람다라는게 파라미터 대신 람다 스코프를 사용해서 가독성을 높여주면서 의도를 명확하게 쓰기 위함이거든요~~
```suggestion
@Composable
fun MainScreen(
    navigator: MainNavigator = rememberMainNavigator()
) {
    Scaffold(
        modifier = Modifier.fillMaxSize(),
        bottomBar = {
            MainBottomBar(
                modifier = Modifier.background(AlarmiTheme.colors.grey800),
                isVisible = navigator.showBottomNavigator(),
                tabs = MainTabType.entries.toPersistentList(),
                currentTab = navigator.currentTab,
                onTabSelected = navigator::navigate
            )
        }
    ) { padding ->
        MainNavHost(
            navigator = navigator,
            padding = padding,
            modifier = Modifier
        )
    }
}
```

요렇게 쓰는게 어떤가요~?

Reply:
넵 반영하겠습니다!
후행람다에 대해 찾아보고 왔는데, UI계층 구조 측면에서 코드의 가독성과 간결성을 극대화할 수 있는 거군뇽 감사합니다~